Launching AMPT fails on Windows clean install or configuration update

If either a configuration directory doesn't exist or if a configuration file is out of date, AMPT needs to copy default configs from the JAR resource to the user's preferences directory.

That is failing on Windows-based OSes with the following error:

```
[ERROR] org.allenai.allenmli.orca.helpers.ConfigurationFileLoadException: Unknown configuration file 'CueConfig.json'
	at org.allenai.allenmli.orca.helpers.ConfigurationLoader.copyDefaultConfigToPath(ConfigurationLoader.java:128)
	at org.allenai.allenmli.orca.helpers.ConfigurationLoader.getFullConfigPath(ConfigurationLoader.java:108)
	at org.allenai.allenmli.orca.helpers.ConfigurationLoader.getJsonFile(ConfigurationLoader.java:186)
	at org.allenai.allenmli.orca.helpers.ConfigurationLoader.getJsonFileAsMap(ConfigurationLoader.java:211)
	at org.allenai.allenmli.orca.ui.CueManager.load_configuration(CueManager.java:77)
	at org.allenai.allenmli.orca.ui.CueManager.<init>(CueManager.java:58)
	at org.allenai.allenmli.orca.ui.ControlWindow.<init>(ControlWindow.java:71)
	at org.allenai.allenmli.orca.AMPT_Main.lambda$run$0(AMPT_Main.java:50)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

--- a/src/main/java/org/allenai/allenmli/orca/helpers/ConfigurationLoader.java
+++ b/src/main/java/org/allenai/allenmli/orca/helpers/ConfigurationLoader.java
@@ -122,7 +122,8 @@ public final class ConfigurationLoader {
   public static void copyDefaultConfigToPath(String filename, Path configurationFile)
       throws ConfigurationFileLoadException {
     final URL resource =
-        ConfigurationLoader.class.getResource(Paths.get(getDefaultConfigDirectory(), filename).toString());
+        ConfigurationLoader.class.getResource(
+            String.format("%s/%s", getDefaultConfigDirectory(), filename));
     if (null == resource) {
       throw new ConfigurationFileLoadException(
           String.format("Unknown configuration file '%s'", filename), null);

